Vikta-UI-testing-sample/src/main/java/com/griddynamics/qa/vikta/uitesting/sample/stepsDefinitions/HomePageSteps.java

```diff
@@ -12,9 +19,47 @@ public HomePageSteps(WebDriver driver) {
     super(driver);
   }
 
-  //TODO: Add more steps.
+  @Step
+  public void openHomePage() {
+    getDriver().get(getData().baseUrl());
+  }
+
+  @Step
+  public void clickSearchButton() {
+    page().clickSearchButton();
+  }
+
+  @Step
+  public String getFirstProductTitle() {
```

Comment:
this should not be a step, because it does not belong to the home page or any other page. Move calling the API methods directly to tests